pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/Application.kt

```diff
@@ -52,7 +67,28 @@ fun main(args: Array<String>) {
  */
 fun Application.module() {
     configureRouting()
```

Comment:
Would you please update the HealthQueryService to selectively check for the health of the messaging service?  In other words, right now it is always checking ASB, but needs to check RabbitMQ, etc depending on the config.